.github/workflows/tests.yaml

```diff
@@ -21,12 +21,20 @@ jobs:
             -   uses: shivammathur/setup-php@v2
                 with:
                     php-version: ${{ matrix.php }}
-                    coverage: none # disable xdebug, pcov
+                    # https://kizu514.com/blog/pcov-is-better-than-phpdbg-and-xdebug-for-code-coverage/
+                    coverage: pcov
 
             -   run: composer update --prefer-lowest --no-progress --ansi
                 if: "matrix.dependencies == 'lowest'"
 
             -   run: composer install --no-progress --ansi
                 if: "matrix.dependencies != 'lowest'"
 
-            -   run: vendor/bin/phpunit
+            -   run: |
+                    vendor/bin/phpunit --coverage-clover build/logs/clover.xml
+                    # Coveralls.io
+                    composer require --dev php-coveralls/php-coveralls:^2.4
+                    vendor/bin/php-coveralls --coverage_clover=build/logs/clover.xml -v
+                env:
+                    COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+                    COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This will force to run coverage on every PHP version.
If there will be a bug in tests/coverage, there will be always 2 places we have to check. E.g. right now coverage is failing, so test are failing as well.

That's why it's better to have them in standalone workflow, e.g. `test_coverage.yaml`

Reply:
Ok, I will try apply to test_coverage.yaml